5/Dockerfile

```diff
@@ -39,7 +39,7 @@ ENV GCC_MIRRORS \
 
 # Last Modified: 2017-10-10
 ENV GCC_VERSION 5.5.0
-# Docker EOL: 2018-10-10
+# Docker EOL: 2019-04-10
```

Comment:
As much as I'd love to see the 5.x series go away instead, I don't think it's entirely reasonable for us to assume there won't be another 5.x release just yet. :sweat_smile: :disappointed: